activemodel/test/cases/dirty_test.rb

```diff
@@ -237,4 +238,9 @@ def save
   test "model can be dup-ed without Attributes" do
     assert @model.dup
   end
+
+  test "to_json should work on model" do
+    @model.name = "Dmitry"
+    assert_equal "{\"name\":\"Dmitry\",\"color\":null,\"size\":null,\"status\":\"initialized\"}", @model.to_json
```

Comment:
Should also test `to_json` with `:except` option.

Reply:
Done @kamipo , please review again.